Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'puppet-blacksmith', '>= 4.1.0', {"groups"=>["development"]}
 gem 'beaker', '>= 4.2.0', {"groups"=>["system_tests"]}
 gem 'beaker-docker', {"groups"=>["system_tests"]}
 gem 'beaker-hostgenerator', '>= 1.1.10', {"groups"=>["system_tests"]}
+gem 'beaker-hiera', {"git"=>"https://github.com/ekohl/beaker-hiera", "branch"=>"fix", "groups"=>["system_tests"]}
```

Comment:
I'm not sure how to deal with this. I'll start nagging to get the upstream repo un-achived and the fix merged

Reply:
I'd say this is OK, as it's "only" about testing.

Reply:
Ok, let's accept this for now. There is talk about moving beaker from puppetlabs to Voxpupuli at which point I can solve it cleanly.